docs/e2e.md

```diff
@@ -221,3 +221,158 @@ tests using a specific one. Example usage:
 ```sh
 ./run.sh -config my_config.yaml
 ```
+
+### Step-by-step DR workflow testing
+
+For manual testing and debugging, you can run individual DR operations step by step.
+This approach is useful when you need to:
+
+- Debug issues at specific stages of the DR workflow
+- Manually verify the state between DR operations
+- Test partial DR flows for development purposes
+- Understand the detailed behavior of each DR operation
+
+> [!IMPORTANT]
+> The step-by-step workflow requires manual cleanup if stopped mid-process.
+> Always ensure you complete the full workflow or manually clean up resources.
+
+### Prerequisites
+
+Before running step-by-step tests, ensure your clusters are properly configured:
+
+1. **Deploy a channel** pointing to the application repository:
+
+   ```sh
+   kubectl apply -k https://github.com/RamenDR/ocm-ramen-samples.git/channel?ref=main --context hub
+   ```
+
+1. **Configure your environment** with the e2e config file as described in the
+   [configuration section](#preparing-a-configyaml-file).
```

Comment:
This is not really requirement of step-by-step usage, it is requirement for using e2e. We can simplify by removing the entire Prerequisites section.